### DISCUSSION TITLE
Is there a batch conversion of real files

### DISCUSSION THREAD

**Discussion by JHeLiu** [minio/minio#16060] [Q&A] [answered]
Is there a batch conversion of real files

**minio** 

1. /logo.jpg/xl.meta 
2. /logo1.jpg/xl.meta 
3. /logo2.jpg/xl.meta

**i need**

1. logo.jpg
2. logo1.jpg
3. logo2.jpg

**Comment by harshavardhana** [comment, marked as answer]
You use S3 API to access those objects - looking at the backend and trying to read it yourself is not allowed
